bingo/bingo-elastic/python/bingo_elastic/elastic.py

```diff
@@ -62,11 +62,9 @@ def get_index_name(record: IndigoRecord) -> IndexName:
 def get_record_by_index(
     response: Dict, index: str
 ) -> Union[IndigoRecordMolecule, IndigoRecordReaction]:
-    if index == IndexName.BINGO_MOLECULE.value:
-        return IndigoRecordMolecule(elastic_response=response)
     if index == IndexName.BINGO_REACTION.value:
         return IndigoRecordReaction(elastic_response=response)
-    raise AttributeError(f"Unknown index {index}")
+    return IndigoRecordMolecule(elastic_response=response)
```

Comment:
should we raise error if index isn’t really exists?

Reply:
That's really hard to track "our" index and other random values because "our" index can be any value that was set up as library-name.

Reply:
I have provided some solution, check please.